### PR TITLE
feat(provider/kuberenetes): V2 deployments

### PR DESCRIPTION
**Pull request by lwander**
1st example of a manifest deploy that triggers a patch op

### CODE REVIEW

File: clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/KubernetesManifestDeployer.java [hunk 1]
```diff
@@ -78,6 +82,8 @@ private KubernetesDeployer findDeployer(KubernetesKind kind) {
         return serviceDeployer;
       case INGRESS:
         return ingressDeployer;
+      case DEPLOYMENT:
+        return deploymentDeployer;
```
Comment (jtk54) [inline clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/KubernetesManifestDeployer.java]: Shoulda named it `Actuator`...

**Comment by jtk54** [comment]
Needs a little rebase.

File: clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/KubernetesManifestDeployer.java (hunk 1 above)
Comment (lwander) [reply clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/KubernetesManifestDeployer.java]: Oh man you're right...

Comment (lwander) [reply clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/KubernetesManifestDeployer.java]: Or `Deployinator`

Comment (jtk54) [reply clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/KubernetesManifestDeployer.java]: `Upstaller`

**Pull request merged**